client/components/search-themes/index.tsx

```diff
@@ -1,184 +1,25 @@
-import { Gridicon } from '@automattic/components';
-import { __experimentalUseFocusOutside as useFocusOutside } from '@wordpress/compose';
-import classnames from 'classnames';
 import { useTranslate } from 'i18n-calypso';
-import { useRef, useState, useEffect } from 'react';
-import KeyedSuggestions from 'calypso/components/keyed-suggestions';
 import Search, { SEARCH_MODE_ON_ENTER } from 'calypso/components/search';
-import { useSelector } from 'calypso/state';
-import { getThemeFilters } from 'calypso/state/themes/selectors';
-import { filterDelistedTaxonomyTermSlugs } from 'calypso/state/themes/utils';
-import { allowSomeThemeFilters, computeEditedSearchElement, insertSuggestion } from './utils';
-import type { ThemeFilters } from './types';
 import './style.scss';
 
 interface SearchThemesProps {
 	query: string;
 	onSearch: ( query: string ) => void;
-	recordTracksEvent: ( eventName: string, eventProperties?: object ) => void;
```

Comment:
I'm counting 7 different `recordTracksEvent` in the deleted side of the diff.
Is this intentional?

Reply:
FWIW, I got the same tracks requests on wp.com as on calypso live when testing.

Reply:
Yes, it's intentional. If you follow the property in the code, you can see that it was only used to track when a user clicked on clearing the search.

Reply:
👍